components/auth/Join.tsx

```diff
@@ -92,6 +95,21 @@ const Join = () => {
           error={formik.touched.password ? formik.errors.password : undefined}
           onChange={formik.handleChange}
         />
+
+
+        {/* ADDING CHECKBOX HERE */}
+        <InputWithCheckbox
+          type="checkbox"
+          label={t('termsandconditon')}
+          name="agreeToTerms"
+          placeholder={t('termsandcondition')}
```

Comment:
```suggestion
          placeholder={t('terms-and-conditon')}
```

Reply:
The word `Terms and conditions` should be a hyperlink that points to `/terms-condition`. Add an empty page for now.